pkg/app/piped/cloudprovider/kubernetes/kubectl.go

```diff
@@ -70,6 +76,74 @@ func (c *Kubectl) Apply(ctx context.Context, namespace string, manifest Manifest
 	return nil
 }
 
+func (c *Kubectl) Create(ctx context.Context, namespace string, manifest Manifest) (err error) {
+	defer func() {
+		kubernetesmetrics.IncKubectlCallsCounter(
+			c.version,
+			kubernetesmetrics.LabelCreateCommand,
+			err == nil,
+		)
+	}()
+
+	data, err := manifest.YamlBytes()
+	if err != nil {
+		return err
+	}
+
+	args := make([]string, 0, 5)
```

Comment:
I think 4 is enough.

```suggestion
	args := make([]string, 0, 4)
```

Reply:
I think it's necessary like `-n <namespace> create -f -`

Reply:
Oops. You are right. 👍